packages/lucia/src/middleware/index.ts

```diff
@@ -103,7 +103,7 @@ export const fastify = (): Middleware<[FastifyRequest, FastifyReply]> => {
 type SvelteKitRequestEvent = {
 	request: Request;
 	cookies: {
-		set: (name: string, value: string, options?: CookieAttributes) => void;
+		set: (name: string, value: string, options: CookieAttributes & { path: string }) => void;
```

Comment:
I'm holding off this PR for now since this would break compatibility with SvelteKit v1 types

Reply:
Compatibility should be OK, unless I'm missing something. Since SvelteKit v1 uses `options?: CookieAttributes`, it'll still be valid when we want `options: CookieAttributes & { path: string }`, which is more restrictive (requiring it to be present with the `path`).

Just to make sure, I tried the following on SvelteKit `1.30.3` with the new Lucia types and it compiles properly:

```typescript
type SvelteKitEvent = RequestEvent<Partial<Record<string, string>>, string | null>;
type LuciaEvent = SvelteKitRequestEvent;

const event = (null! as SvelteKitEvent) satisfies LuciaEvent;
```

There's also no type error when passing a SvelteKit v1 event to `handleRequest`:

```typescript
const handleAuth: Handle = async ({ event, resolve }) => {
	event.locals.auth = auth.handleRequest(event);
	return await resolve(event);
};
```